prs_per_repo.rb

```diff
@@ -49,10 +49,17 @@ def repo_in_org?(repo, org)
   end
 
   def execute_query(query)
-    results = stats.client.search_issues(query)
-puts "github query=#{query.inspect} returned #{results.total_count} items"
-#puts "query results=#{results.inspect}"
-    results.items
+    begin
+      results = stats.client.search_issues(query)
+      puts "GitHub query=#{query.inspect} returned #{results.total_count} items"
+      #puts "query results=#{results.inspect}"
+      results.items
+    rescue Octokit::TooManyRequests
+      retry_time = 5
+      puts "GitHub API rate limit exceeded.  Retrying in #{retry_time} seconds."
```

Comment:
You have to wait until the top of the hour, so if you calculate the number of seconds until the top of the hour, then there's no need to poll.

Reply:
It is the [GitHub Search API Rate Limit](https://developer.github.com/v3/search/#rate-limit) which allows up to 30 requests a minute.

Reply:
oh! didn't know this was a separate limit.